DOCUMENTATION.md

```diff
@@ -0,0 +1,121 @@
+# Documentation
+
+## Flatseal permissions
+
+This section indicates the permissions of Flatpak with better integration with Flatseal. It is based on the official [Sandbox Permissions Reference](https://docs.flatpak.org/en/latest/sandbox-permissions-reference.html).
+
+Flatseal follows the same concept as `flatpak-override`: overriding the default permissions that the Flatpak program originally shipped with. Every change the user does will trigger Flatseal to output the overriden permissions in the `~/.local/share/flatpak/overrides` directory.
+
+If you want to read more into Flatseal's permissions, you can look at the [Permissions section](#permissions).
+
+If you want to read more into `flatpak override`, you can look at the [`FLATPAK OVERRIDE(1)` manpage](https://manpages.debian.org/testing/flatpak/flatpak-override.1.en.html).
```

Comment:
It seems debian's man page is outdated, better use Flatpak's version https://docs.flatpak.org/en/latest/flatpak-command-reference.html#flatpak-override

Reply:
https://github.com/tchx84/Flatseal/pull/136/commits/1d7d6b699f59f1f56cb7e7301bec3ada021ad0de